core_types/src/error/mod.rs

```diff
@@ -118,6 +118,7 @@ pub enum HolochainError {
     DnaHashMismatch(HashString, HashString),
     EntryNotFoundLocally,
     EntryIsPrivate,
+    // MutexWouldBlock,
```

Comment:
still need this commented variant?

Reply:
Guess I'll remove that, it was a half-implemented idea